Kimai2.grandtotalplugin/index.js

```diff
@@ -145,11 +145,15 @@ function kimaiGetTimesheets(url, username, token)
 
     for (var aEntry of aItems)
     {
+        var minutes = Math.round(aEntry['duration'] / 60);
+        if (Math.round(aEntry['rate']/minutes*60,2)!=aEntry['rate']/minutes*60) {
+            aEntry['rate'] = minutes / 60 * Math.round(aEntry['rate']/minutes*60,2);
+        }
```

Comment:
```suggestion
        var roundedMinutes = aEntry['rate'] / minutes * 60;
        if (Math.round(roundedMinutes, 2) != roundedMinutes) {
            aEntry['rate'] = minutes / 60 * Math.round(roundedMinutes, 2);
        }
```

Reply:
Here is the result with your calculation:

![Screenshot 2023-12-13 at 12 23 52](https://github.com/Keleo/kimai2-grandtotal/assets/46503492/e2f2c0fa-bc27-4a29-a77d-ea69be9d3644)

Reply:
Nah, my changes are really only code styles and one extracted variable.